tools/cpp/osx_cc_configure.bzl

```diff
@@ -25,30 +25,8 @@ load(
     "configure_unix_toolchain",
     "find_cc",
     "get_env",
-    "get_escaped_cxx_inc_directories",
 )
 
-def _get_escaped_xcode_cxx_inc_directories(repository_ctx, cc, xcode_toolchains):
-    """Compute the list of default C++ include paths on Xcode-enabled darwin.
-
-    Args:
-      repository_ctx: The repository context.
-      cc: The default C++ compiler on the local system.
```

Comment:
Can this ever be a cc from a locally built toolchain? For example if I build clang locally and want to use that, it seems that this change prevents me from doing that.

Reply:
Correct. Also, clang could be installed in `/Library/Developer/CommandLineTools` which isn't covered by this new stricter check.